### PR TITLE
getting involved with urbit

### PR DESCRIPTION
**Pull request by gordoncc**
link list for @thelifeandtimes and @nealdavis's talk at assembly

### CODE REVIEW

**Comment by gordoncc** [comment]
New page at /getinvolved

File: pages/getinvolved.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      </Head>
+      <IntraNav ourSite="https://urbit.org" search={search} />
+      <SingleColumn>
+        <Header search={search} />
+        <Section>
+          <h1 className="pb-16">Getting Started With Urbit</h1>
+          <div className="">
+
+          <p className="text-md">
+	         • <Link href="https://developers.urbit.org/courses">Take a course on hoon</Link>
```
Comment (sigilante) [inline pages/getinvolved.js]: Capitalize Hoon

**Review by sigilante — CHANGES_REQUESTED** [review]
Comments.

File: pages/getinvolved.js (hunk 1 above)
Comment (sigilante) [reply pages/getinvolved.js]: Maybe add "Build an app." link to lightning guides?

**Review by sigilante — APPROVED** [review]

**Comment by thelifeandtimes** [comment]
lgtm, merging.

**Pull request merged**